editions/tw5.com/tiddlers/concepts/WikiText.tid

```diff
@@ -1,15 +1,20 @@
 created: 20131205155227468
-modified: 20140919191220377
+custom-list: [[Headings in WikiText]] [[Paragraphs in WikiText]] [[Hard Linebreaks in WikiText]] [[Formatting in WikiText]] [[Dashes in WikiText]] [[Block Quotes in WikiText]] [[Lists in WikiText]] [[Definitions in WikiText]] [[Horizontal Rules in WikiText]] [[Tables in WikiText]] [[Code Blocks in WikiText]] [[Typed Blocks in WikiText]] [[Images in WikiText]] [[Linking in WikiText]] [[Variables in WikiText]] [[Macro Definitions in WikiText]] [[Macro Calls in WikiText]] [[Macros in WikiText]] [[Transclusion in WikiText]] [[Transclusion and Substitution]] [[Styles and Classes in WikiText]] [[HTML in WikiText]] [[Widgets in WikiText]] [[WikiText Parser Modes]]
+modified: 20220727095329372
 tags: Concepts Reference
 title: WikiText
 type: text/vnd.tiddlywiki
 
-~WikiText is a concise, expressive way of typing a wide range of text formatting, hypertext and interactive features. It allows you to focus on writing without a complex user interface getting in the way. It is designed to be familiar for users of [[MarkDown|http://daringfireball.net/projects/markdown/]], but with more of a focus on linking and the interactive features.
+~WikiText is a concise, expressive way of typing a wide range of text formatting, hypertext and interactive features. It allows you to focus on writing without a complex user interface getting in the way. It is designed to be familiar for users of MarkDown but with more of a focus on linking and the interactive features.
 
 ~WikiText can also be inserted to the text field using the [[Editor toolbar]].
 
-See [[Formatting text in TiddlyWiki]] for an introduction to WikiText.
+The following list of syntax rules ''start simple and get more complex'' at the end of the list.
 
-The following elements of WikiText syntax are built into the core:
+<$button actions=<<wikitext-open-custom-list>> >Open all tiddlers from the list below</$button>
 
-<<list-links "[tag[WikiText]]">>
+@@.multi-columns
+<<list-links-draggable tiddler:"WikiText" field:"custom-list">>
```

Comment:
Wouldn't it be more convenient to keep the list inline here? ie:

```
<<list-links-draggable "[[Headings in WikiText]] [[Paragraphs in WikiText]] [[Hard Linebreaks in WikiText]] [[Formatting in WikiText]] [[Dashes in WikiText]] [[Block Quotes in WikiText]] [[Lists in WikiText]] [[Definitions in WikiText]] [[Horizontal Rules in WikiText]] [[Tables in WikiText]] [[Code Blocks in WikiText]] [[Typed Blocks in WikiText]] [[Images in WikiText]] [[Linking in WikiText]] [[Variables in WikiText]] [[Macro Definitions in WikiText]] [[Macro Calls in WikiText]] [[Macros in WikiText]] [[Transclusion in WikiText]] [[Transclusion and Substitution]] [[Styles and Classes in WikiText]] [[HTML in WikiText]] [[Widgets in WikiText]] [[WikiText Parser Modes]]">>
```

Reply:
Not really, since the "open all" button also needs it.

Reply:
But I am suggesting that we remove the "open all" button

I reiterate that my goal here is to try to keep the docs as internally consistent as possible.

Reply:
[list-links-draggable](https://tiddlywiki.com/prerelease/#list-links-draggable%20Macro) needs a "place" to store the results. If new elements may be added in the future, we can simply "drag & drop" them to the list. .. Or the list can be re-sorted, if it turns out the order isn't 100% right yet.